CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Removed transitive (provided) dependency on `hbase-client`.
 ### Fixed
 * Removed 'isOpen()' method from CloseableMetaStoreClient. See [#11](https://github.com/HotelsDotCom/hcommon-hive-metastore/issues/11).
+* Issue where the wrong exception was being propagated in the compatibility layer.
```

Comment:
Could you create a quick GH issue with the relevant part of the stack and link it here? Just so in future we have a reference to this with some more detail.

Reply:
https://github.com/HotelsDotCom/hcommon-hive-metastore/issues/16